d2exporter/export.go

```diff
@@ -45,6 +45,8 @@ func applyStyles(shape *d2target.Shape, obj *d2graph.Object) {
 	}
 	if obj.Attributes.Style.Fill != nil {
 		shape.Fill = obj.Attributes.Style.Fill.Value
+	} else if obj.Attributes.Shape.Value == d2target.ShapeText {
```

Comment:
This is ok, but I wonder if it makes sense to have this as a default value all the way since parsing..

Reply:
-1, i think the paradigm of compiler ingesting user inputs and then export applying defaults is right